.ci/install/isal.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Same feedback as from #117

Reply:
Changed the mirrors. Didn't see that comment until now. I think the script is executable. Will chmod on vagrant (since i'm on windows) to ensure!